msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/MessagePackGenerator.java

```diff
@@ -109,14 +108,7 @@ public MessagePackGenerator(int features, ObjectCodec codec, OutputStream out)
         }
         messageBufferOutputHolder.set(messageBufferOutput);
 
-        if (messagePacker == null) {
-            messagePacker = MessagePack.newDefaultPacker(messageBufferOutput);
-        }
-        else {
-            messagePacker.reset(messageBufferOutput);
```

Comment:
Right. The actual cost here is just an object instantiation cost. And it could be negligible if we serialize ~ 10,000 objects. 

@komamitsu What do you think?